README.rst

```diff
@@ -43,6 +43,15 @@ This exporter takes the Draft.js ContentState data as input, and outputs HTML ba
 
     pip install draftjs_exporter
 
+You will also need to install a backing engine. The exporter supports html5lib via BeautifulSoup, and lxml. lxml is the most performant, but it requires ``libxml2`` and `libxslt`` to be available on your system.
+
+.. code:: sh
+
+    # For html5lib,
+    pip install 'html5lib>=0.999,<=1.0b10' 'beautifulsoup4>=4.4.1,<5'
+    # For lxml,
+    pip install lxml
```

Comment:
We should use `extras_require`, the same way [requests does it](https://github.com/kennethreitz/requests/blob/524627ef21818e06a33bf4b54cd9c93a7dc35b21/setup.py#L97-L100), so we could instruct doing `pip install draftjs_exporter[html5lib]` or `pip install draftjs_exporter[lxml]` which allow us to control the dependencies versions and make everybody's `requirement.txt` file much simpler and clearer.